R/JasmineLai/edit_otherEntity_EML.R

```diff
@@ -0,0 +1,57 @@
+## Get token!!!
+# options(dataone_test_token = "...")
+
+# load packages
+library(dataone)
+library(datapack)
+library(EML)
+library(remotes)
+library(XML)
+library(arcticdatautils)
+library(datamgmt)
+
+# setting node to arctic node
+cn <- CNode("PROD")
+adc <- getMNode(cn, "urn:node:ARCTIC") # the mn
+
+# making sure that we have to most recent reasource map
+rm_pid_original <- "resource_map_urn:uuid:9e7325b5-a344-499b-99c1-aa3421f2d297"
+all_rm_versions <- get_all_versions(adc, rm_pid_original)
+rm_pid <- all_rm_versions[length(all_rm_versions)]
+
+pkg <- get_package(adc,
+  rm_pid,
+  file_names = T
+)
+
+# grab the eml
+doc <- read_eml(getObject(adc, pkg$metadata))
+
+# Updating the entityDescription in otherEntity, LIDAR
+
+# get the index of the LIDAR otherEntity and save to to be used
+# searches the entityDescriptions for one with LIDAR in it
+num <- which_in_eml(
```

Comment:
can also use `arcticdatautils::eml_get_simple()` here to grab a list of entity names. I  use a workflow like this frequently:

```
ent_names <- eml_get_simple(doc$dataset$otherEntity, "entityName")
i <- which(ent_names == "LIDAR.zip")
```

Reply:
thanks!

i like this workflow better... fewer mental hoops to jump through to make the right function

I'll probably try to get the `eml_get_simple()` mentioned in the training manual so new users know of its' existence

Reply:
yeah thats a great idea. `which_in_eml` is only kind of helpful for me, because of the hoops you mention :) I rarely use it